ChangeLog.md

```diff
@@ -67,6 +67,8 @@
 
   * Fix bug in `pl-symbolic-input` to handle submissions that simplify to invalid expressions (Tim Bretl).
 
+  * Fix bug in `pl-symbolic-input` to handle the sympy constants I and E properly (Tim Bretl).
+
```

Comment:
Add a second ChangeLog line something like "Add `allow-complex` attribute for `pl-symbolic-input`"?

Reply:
done